src/PresentationalComponents/ZeroState/ZeroState.js

```diff
@@ -30,6 +30,8 @@ import InsightsLink from '@redhat-cloud-services/frontend-components/InsightsLin
 
 const ZeroState = () => {
     const intl = useIntl();
+    const getEnvironment = useChrome();
+    const isItLessEnv = getEnvironment() === 'ephem' || 'int' || 'frhStage' || 'frh';
```

Comment:
Also, why it is required to change the button URL for ephemeral environments?

Reply:
sorry about the lag here, was out for a week but I switched to using a FF instead.